hack/setup-build-index-image.sh

```diff
@@ -90,8 +90,10 @@ spec:
           cd ..
 
 
-          git clone --single-branch --branch GATEKEEPER_VERSION https://github.com/open-cluster-management/gatekeeper-operator.git
+          git clone --single-branch --branch GATEKEEPER_VERSION https://github.com/stolostron/gatekeeper-operator.git
           cd gatekeeper-operator
+          # replace u/s image with d/s
+          sed -i 's/quay.io\/gatekeeper\/gatekeeper-operator/registry.redhat.io\/rhacm2\/gatekeeper-rhel8-operator/g' bundle/manifests/gatekeeper-operator.clusterserviceversion.yaml
```

Comment:
I don't think we should do something like that here

Reply:
I don't think we need this change anymore. I did that to fix ci errors, but the issue was resolved, and anyway we are checking if gatekeeper should still be tested, so maybe we shouldn't touch or change is now. I'm closing this PR.